ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -390,30 +387,12 @@ class WalletHomeScreen extends React.PureComponent<Props, State> {
     );
   }
 
-  private renderHelpMessage = (
-    alignCenter: boolean = false
-  ): React.ReactNode => (
-    <React.Fragment>
-      <VSpacer size={24} />
-      <Body style={alignCenter ? styles.centered : undefined}>
-        {`${I18n.t("wallet.transactionHelpMessage.text1")} `}
-        <Body
-          weight={"SemiBold"}
-          style={alignCenter ? styles.centered : undefined}
-        >
-          {I18n.t("wallet.transactionHelpMessage.text2")}
-        </Body>
-      </Body>
-    </React.Fragment>
-  );
-
-  private transactionError(renderHelp: boolean) {
+  private transactionError() {
     return (
       <Content
         scrollEnabled={false}
         style={[styles.noBottomPadding, styles.whiteBg, IOStyles.flex]}
       >
-        {renderHelp && this.renderHelpMessage()}
         <VSpacer size={24} />
```

Comment:
now that there is no text to space before that spacer, I believe this could be removed, what do you think?
also, to me that bonus spacing looks weird without any text 😅

Reply:
Good catch! 

Addressed in 91db2b1dd2cfbe367e56c098f9b03bd7a6267377